lib/grpc/endpoint.ex

```diff
@@ -94,12 +94,29 @@ defmodule GRPC.Endpoint do
     server_interceptors(tail, acc)
   end
 
-  defp parse_servers(servers) do
+  defp parse_servers(servers, false) do
+    servers = [{GRPC.Reflection.Service, %{}}] ++ servers
+
     servers
     |> Enum.map(fn {server, _} -> server end)
     |> List.flatten()
   end
 
+  defp parse_servers(servers, true) do
+    servers
+    |> Enum.map(fn {server, _} -> server end)
+    |> List.flatten()
+  end
```

Comment:
To avoid duplication, WDYT?
```suggestion
  defp parse_servers(servers) do
    case reflection_enabled?() do
      true -> [{GRPC.Reflection.Service, %{}} | servers]
      false -> servers
    end
    |> Enum.map(fn {server, _} -> server end)
    |> List.flatten()
  end
```